docs/administration.md

```diff
@@ -23,10 +23,10 @@ The admin module provides the following functions:
 ## Activate/deactivate/decommission resources
 
 ### What is deactivation of a resource?
-Deactivation refers to a temporary shutdown. This means that the resource will not be available for use unless and until it is activated later through the admin portal as required by the country.
+Deactivation refers to a reversible action in which the isActive value for a resouce is database is set to "False". This means that the resource will not be available for use unless and until it is activated later through the admin portal as required by the country.
```

Comment:
spell check "resource"

Reply:
Thanks @anusha , corrected the spelling, pls approve/merge